db/view/view_update_generator.cc

```diff
@@ -179,6 +179,8 @@ future<> view_update_generator::start() {
                     // Need to add sstables back to the set so we can retry later. By now it may
                     // have had other updates.
                     std::move(sstables.begin(), sstables.end(), std::back_inserter(_sstables_with_tables[t]));
+                    // Sleep a bit, to avoid a tight loop repeatedly spamming the log with the same message.
+                    seastar::sleep(std::chrono::seconds(1)).get();
```

Comment:
I think that @kostja wanted to standardize on 0.1 seconds (100ms) sleeps.
Regarding the log spam, shouldn't we use `logger::rate_limit`?

That said, if you think that 1s sleep is more appropriate for this use case, I'm good with that.

Reply:
Where did he want to standardized this way?
In my response to Botond above, I tried to explain why I think it doesn't matter what the sleep is (within reason), I don't think it needs to be smart or even standardized. It's not something that a user will ever encounter (it's not a user-facing timeout or delay or anything). The only purpose of this sleep is to reduce log spam - from a million lines per second to just 1 per second - 0.1 or 10 per second would also be just as acceptable.